packages/monorepo-scripts/src/postpublish_utils.ts

```diff
@@ -93,8 +93,10 @@ export const postpublishUtils = {
         const notes = this.getReleaseNotes(packageName, version);
         const releaseName = this.getReleaseName(packageName, version);
         const tag = this.getTag(packageName, version);
+        // tslint:disable-next-line:no-unused-variable
```

Comment:
Why are so many of these necessary?

Reply:
I can just wrap them in a `disable` comment (or remove the var and disable the no-dangling promises rule)

Reply:
So to clarify: I am being extremely conservative with removing. I'm only deleting the expression if it's a simple assignment, or if it's a read-only assignment. If the expression could have a side effect, or if it is a private method on a class (that could be used in tests or elsewhere) I just add the tslint comment. 

Not sure if your comment here refers to this file in particular (which I can clean up) or more generally.

Here is an example of a private method that we use pretty freely for some reason.

```js
src/contract_wrappers.ts:        (this.exchange as any)._setNetworkId(networkId);
src/contract_wrappers.ts:        (this.tokenRegistry as any)._setNetworkId(networkId);
src/contract_wrappers.ts:        (this.token as any)._setNetworkId(networkId);
src/contract_wrappers.ts:        (this.proxy as any)._setNetworkId(networkId);
src/contract_wrappers.ts:        (this.etherToken as any)._setNetworkId(networkId);
src/contract_wrappers/contract_wrapper.ts:    private _setNetworkId(networkId: number): void {
```